CODEOWNERS

```diff
@@ -16,6 +16,19 @@
 /aten/src/ATen/native/native_functions.yaml @bdhirsh
 /aten/src/ATen/native/tags.yaml @anjali411
 
+# Architecture Optimization (quantization, sparsity, etc.)
+/torch/quantization/ @jerryzh168 @vkuzo
+/test/test_quantization.py @jerryzh168
+/test/ao/ @jerryzh168 @z-a-f @hdcharles
+/test/quantization/ @jerryzh168 @z-a-f
+nn/quantized/ @jerryzh168 @vkuzo
+nn/quantizable/ @jerryzh168 @z-a-f
+nn/qat/ @jerryzh168 @vkuzo
+nn/intrinsic @jerryzh168 @vkuzo
+ao/ @vkuzo
```

Comment:
can you add ao/quantization and add my name as well

Reply:
and why this does not start with `/`?

Reply:
>and why this does not start with /?

This is the same as `**/ao/*` basically. So anything that has `/ao/` in its path will get added.